pages/index.js

```diff
@@ -410,30 +410,25 @@ export default function Home({ total, today }) {
                 {features.map((feature) => (
                   <div key={feature.name} className="pt-6">
                     <div className="flow-root rounded-lg bg-gray-50 px-6 pb-8">
-                      <div className="-mt-6">
-                        <div>
-                        <Link aria-label="Go to feature page"
+                      <Link aria-label="Go to ${feature.name} page"
                                href={feature.path} >
-                          <span className="inline-flex items-center justify-center rounded-xl bg-indigo-500 p-3 shadow-lg">
-                            <feature.icon
-                              className="h-8 w-8 text-white"
-                              aria-hidden="true"
-                            />    
-                          </span>
-                        </Link>
-                        </div>
-                        <h3 className="mt-8 text-lg font-semibold leading-8 tracking-tight">
-                          <Link
-                            href={feature.path}
-                            className="text-gray-900 hover:text-indigo-600 hover:underline"
-                          >
+                        <div className="-mt-6">
+                          <div>
+                            <span className="inline-flex items-center justify-center rounded-xl bg-indigo-500 p-3 shadow-lg">
+                              <feature.icon
+                               className="h-8 w-8 text-white"
+                               aria-hidden="true"
+                               />    
+                            </span>
+                          </div>
+                        <h3 className="mt-8 text-lg font-semibold leading-8 tracking-tight text-gray-900 hover:text-indigo-600 hover:underline">
```

Comment:
we should also change the hover from h3 to the card by giving the root div a `group` class
and change the h3 hovers to this: `group-hover:text-indigo-600 group-hover:underline`